integreat_cms/static/src/js/dashboard.ts

```diff
@@ -0,0 +1,8 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const mobile = document.querySelector("#mobile-menu");
+    const sidebar = document.querySelector("#primary-navigation");
+
+    mobile.addEventListener("click", () => {
+        sidebar.classList.toggle("-translate-x-full");
+    });
+});
```

Comment:
I think this file is not required anymore

Reply:
Oh yes, thank you!